src/index.js

```diff
@@ -168,6 +173,7 @@ RBSheet.defaultProps = {
   customStyles: {},
   onClose: null,
   onOpen: null,
+  onCloseOnPressMask: null,
```

Comment:
I added a new prop to listen when the user makes click outside the bottom sheet.